tests/vmi_lifecycle_test.go

```diff
@@ -259,11 +259,16 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				Should(ContainSubstring(`"subcomponent":"libvirt"`))
 		})
 
-		DescribeTable("log libvirtd debug logs should be", func(vmiLabels, vmiAnnotations map[string]string, expectDebugLogs bool) {
+		DescribeTable("log libvirtd debug logs should be", func(vmiLabelKey, vmiLabelValue, vmiAnnotationKey, vmiAnnotationValue string, expectDebugLogs bool) {
```

Comment:
I think the test would be a bit prettier if you keep this function's signature. Having to add an `if` line is a hint that something is wrong.

```
for k, v := range vmiLabelValue {
  options = append(options, libvmi.WithLabel(k, v))
}
```

Reply:
Done